spaces/domains.json

```
@@ -1,5 +1,6 @@
 {
   "vote.anarchydao.io": "anarchytoken.eth",
+  "gov.tigris.trade": "tigrisdao.eth"
```

A comma is missing at the end of the line of your sub-domain.
```suggestion
  "gov.tigris.trade": "tigrisdao.eth",
```